.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+
+### Before suggesting a new feature
+
+**Ensure the feature has not already been suggested** by searching on GitHub under [**Issues**](https://github.com/iotaledger/iota.py/issues).
+
+### Suggesting a new feature
+
+To suggest a new feature, talk to the IOTA community and IOTA Foundation members on [Discord](https://discord.iota.org/).
+
+If the team members approves your feature, they will create an issue for it.
+</details>
+
+<br>
+
+<details>
+<summary>Build a new feature :hammer:</summary>
+<br>
+
+This section guides you through building a new feature. Following these guidelines helps give your feature the best chance of being approved and merged.
+
+### Before building a new feature
+
+Make sure to discuss the feature with the developers on [Discord](https://discord.iota.org/).
+
+Otherwise, your feature may not be approved at all.
+
+### Building a new feature
+
+To build a new feature, check out a new branch based on the `master` branch, and be sure to consider the following:
+
+- If the feature has a public facing API, make sure to document it, using [Sphinx](https://www.sphinx-doc.org/en/master/) code comments
+
+</details>
+
+<br>
+
+<details>
+<summary>Contribute to the documentation :black_nib:</summary>
+<br>
+
+The Python client library documentation is hosted on https://docs.iota.org, which is built from content in the [documentation](https://github.com/iotaledger/documentation) repository.
+
+Please see the [guidelines](https://github.com/iotaledger/documentation/CONTRIBUTING.md) on the documentation repository for information on how to contribute to the documentation.
+</details>
+
+<br>
+
+<details>
+<summary>Pull requests :mega:</summary>
+<br>
+
+This section guides you through submitting a pull request (PR). Following these guidelines helps give your PR the best chance of being approved and merged.
+
+### Before submitting a pull request
+
+When creating a pull request, please follow these steps to have your contribution considered by the maintainers:
+
+- A pull request should have only one concern (for example one feature or one bug). If a PR addresses more than one concern, it should be split into two or more PRs.
+
+- A pull request can be merged only if it references an open issue
+
+    **Note:** Minor changes such as fixing a typo can but do not need an open issue.
+
+- All code should be well tested
+
+### Submitting a pull request
+
+The following is a typical workflow for submitting a new pull request:
+
+1. Fork this repository
+2. Create a new branch based on your fork
+3. Commit changes and push them to your fork
+4. Create a pull request against the `master` branch
+
+If all [status checks](https://help.github.com/articles/about-status-checks/) pass, and the maintainer approves the PR, it will be merged.
+
+**Note:** Reviewers may ask you to complete additional work, tests, or other changes before your pull request can be approved and merged.
```

Comment:
```suggestion
**Note:** Reviewers may request changes before your pull request can be approved and merged.
```

💭 "ask you to complete additional work" felt a bit too much like being given an assignment to me.